client/components/Cart/populator.sql

```diff
@@ -0,0 +1,43 @@
+
+psql postgres
+
+\c bytes;
+
+INSERT INTO cart(user_id, product_id, cart_quantity)
+ VALUES (1, 1, 1)
+ ON CONFLICT (user_id, product_id)
+ DO UPDATE SET cart_quantity = cart.cart_quantity + 1;
+
+ INSERT INTO cart(user_id, product_id, cart_quantity)
+ VALUES (1, 2, 2)
+ ON CONFLICT (user_id, product_id)
+ DO UPDATE SET cart_quantity = cart.cart_quantity + 1;
+
+ INSERT INTO cart(user_id, product_id, cart_quantity)
+ VALUES (2, 3, 1)
+ ON CONFLICT (user_id, product_id)
+ DO UPDATE SET cart_quantity = cart.cart_quantity + 1;
+
+ INSERT INTO cart(user_id, product_id, cart_quantity)
+ VALUES (1, 4, 1)
+ ON CONFLICT (user_id, product_id)
+ DO UPDATE SET cart_quantity = cart.cart_quantity + 1;
+
+ INSERT INTO cart(user_id, product_id, cart_quantity)
+ VALUES (1, 5, 2)
+ ON CONFLICT (user_id, product_id)
+ DO UPDATE SET cart_quantity = cart.cart_quantity + 1;
+
+ INSERT INTO cart(user_id, product_id, cart_quantity)
```

Comment:
Looks good. Adding some test data to the cart DB to populate the cart to simulate a users choices.